releasenotes/1.9.2.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: releasenotes
+title: Release Notes for Version 1.9.2
+date: 2016-01-16 12:00:00
+release_notes: true
+permalink: /releasenotes/1.9.2/
+---
+
+1. [Introduction](#intro)
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+You can read the official announcement at [(TODO) Apache jclouds 1.9.2 released](http://markmail.org/message/7axjkmb7kypgvtot). To get started with jclouds, please see the [jclouds installation guide](/start/install/).
```

Comment:
Yes, I was waiting for the mirrors to send the annoincement. I'll update the link in a while.